components/MNModel/index.tsx

```diff
@@ -31,28 +31,19 @@ const Model = (props: Props) => {
 
   return (
     <group dispose={null}>
-      <mesh
```

Comment:
I think Cube and Cube_1 were the way to go

Reply:
bruh